docs/whatsnew/1.2.0/1_2_0.rst

```diff
@@ -5,34 +5,40 @@ Pylake 1.2.0
 
 Here's a sneak peak at some of the highlights from the upcoming Pylake `v1.2.0` release...
 
+Improved binding time analysis
+------------------------------
+
 Correctly use the minimum observable time in dwell time analysis
-----------------------------------------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In earlier versions of Pylake, binding time analysis would return biased estimates when analyzing kymographs with very few events.
-This was because `KymoTrackGroup.fit_binding_times()` relied on the assumption that the shortest track in the `KymoTrackGroup` represented the minimum observable dwell time.
-This assumption is likely valid for kymographs with many tracks but problematic when few events occur per kymograph. In this case, binding times will be underestimated.
-For more information see the :doc:`changelog</changelog>`.
+This was because :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.fit_binding_times()` relied on the assumption that the shortest track in the
+:class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` represented the minimum observable dwell time.
+This assumption is likely valid for kymographs with many tracks but problematic when few events occur per kymograph. In this case, binding
+times will be underestimated. For more information see the :doc:`changelog</changelog>`.
 
 .. important::
 
     The old (incorrect) behavior is maintained as default until the next major release (`v2.0.0`) to ensure
     backward compatibility. To enable the fixed behavior, specify `observed_minimum=False` when calling
-    `KymoTrackGroup.fit_binding_times()`.
+    :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.fit_binding_times()`.
 
     Note that CSVs exported from the kymotracker widget  before `v1.2.0` will contain insufficient metadata
-    to make use of the improved analysis. To create this metadata, use `lk.filter_tracks()` on the group with a specified `min_length` before further analysis.
+    to make use of the improved analysis. To create this metadata, use :func:`~lumicks.pylake.filter_tracks()` on the group with a specified
+    `min_length` before further analysis.
 
 Correct for discretization in binding lifetime analysis
--------------------------------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can now take into account the discretized nature of observed binding times of tracked particles. In some cases, this can significantly improve results by removing bias from the fitted lifetimes.
-Simply use `discrete_model=True` with :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.fit_binding_times()`.
+You can now take into account the discretized nature of observed binding times of tracked particles. In some cases, this can significantly
+improve results by removing bias from the fitted lifetimes. Simply use `discrete_model=True` with
```

Comment:
I wonder if it would help to make "some cases" more concrete perhaps by mentioning that especially short dwelltimes (compared to the line time) benefit from this.